views/updateBrowser/updateBrowser.html

```diff
@@ -0,0 +1,18 @@
+<div class="jw-cover jw-modal ng-cloak">
+    <div class="jw-modal-container">
+        <div class="jw-modal-image">
+            <img class="jw-header-logo" ng-src="{{ ::vm.config.assets.banner }}" alt="logo"/>
+        </div>
+        <div class="jw-modal-message jw-modal-message-title">
+            Your browser is out-of-date.
+        </div>
+        <div class="jw-modal-message jw-modal-message-small">
+            Update your browser to view this website correctly.
```

Comment:
Update your current browser, or install one of the other compatible browsers to fully enjoy the JW Showcase experience.

Reply:
agree and done